### PR TITLE
Revert accidentally introduced port change 

### PR DESCRIPTION
**Pull request by brandshaide**

### CODE REVIEW

File: Documentation/contributing/testing/documentation.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ real-time preview. It relies on the ``cilium/docs-builder`` Docker container.
 
     $ make render-docs-live-preview
 
-and preview the documentation at http://localhost:9081/ as you make changes. After making changes to
+and preview the documentation at http://localhost:9082/ as you make changes. After making changes to
```
Comment (pchaigno) [inline Documentation/contributing/testing/documentation.rst]: @aanm Was that port changed by mistake in cce90934c0b3c0c938cddc38241a00618774965f? Seems like something you may have done to temporarily resolve a port conflict (e.g., dev. VM vs. render-docs).

Comment (aanm) [reply Documentation/contributing/testing/documentation.rst]: Ups, yes, that change should have never been pushed.

Comment (pchaigno) [reply Documentation/contributing/testing/documentation.rst]: Maybe safer to just revert the port change in that case?

Comment (aanm) [reply Documentation/contributing/testing/documentation.rst]: yes, @brandshaide can you revert the port accidentally changes in the commit point out by Paul? Thanks

Comment (brandshaide) [reply Documentation/contributing/testing/documentation.rst]: > yes, @brandshaide can you revert the port accidentally changes in the commit point out by Paul? Thanks

@aanm of course, I'll take care of it 🙂

Comment (brandshaide) [reply Documentation/contributing/testing/documentation.rst]: @aanm @pchaigno done

**Review by pchaigno — CHANGES_REQUESTED** [review]
Thanks @brandshaide! Could you squash the two commits please?

**Comment by brandshaide** [comment]
> Thanks @brandshaide! Could you squash the two commits please?

@pchaigno yup, of course. done.

**Review by pchaigno — APPROVED** [review]

**Review by aanm — APPROVED** [review]
Thanks!

**Pull request merged**